manifests/0000_70_cluster-network-operator_03_deployment.yaml

```diff
@@ -113,6 +113,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        - name: OVN_EGRESSIP_HEALTHCHECK_PORT
+          value: "29110"
```

Comment:
That port will be blocked by our current firewall rules on cloud environments:
https://github.com/openshift/enhancements/blob/master/dev-guide/host-port-registry.md

Reply:
Probably pick something that's free in the range 9000-9999 (see above document) and have that port reserved internally (via that document?) so that other services will not use it.

Reply:
Yeah!!! TY for the pointer

Reply:
Using port 9107:  https://github.com/openshift/enhancements/pull/1209